index.bs

```diff
@@ -2862,7 +2860,7 @@ partial interface MLGraphBuilder {
     **Arguments:**
         - *input*: an {{MLOperand}}. The input N-D tensor.
         - *options*: an optional {{MLTransposeOptions}}. The optional parameters of the operation.
-            - *permutation*: a sequence of {{long}} values. The values used to permute the output shape. When it's not specified, it's set to `[N-1...0]`, where `N` is the rank of the input tensor. These default values cause the output to become a transposed tensor of the input. When specified, the number of values in the sequence must be the same as the rank of the input tensor, and the values in the sequence must be within the range from 0 to N-1 with no two or more same values found in the sequence.
+            - *permutation*: a sequence of {{unsigned long}} values. The values used to permute the output shape. When it's not specified, it's set to `[N-1...0]`, where `N` is the rank of the input tensor. These default values cause the output to become a transposed tensor of the input. When specified, the number of values in the sequence must be the same as the rank of the input tensor, and the values in the sequence must be within the range from 0 to N-1 with no two or more same values found in the sequence.
```

Comment:
How about adding commas here `[N-1, ..., 0]` and/or adding a tiny example? "When it's not specified, it's set to `[N-1...0]`, where `N` is the rank of the input tensor (e.g. [2,1,0] for a 3D tensor)."

Reply:
sgtm. I applied this proposal to `MLTransposeOptions.permutation` and also made `MLSliceOptions.axes` aligned in latest commit https://github.com/webmachinelearning/webnn/pull/359/commits/ef94c4e4c74d0f9087e59b567300c925b616f253. Please take another look. Thanks!